Uchu.StandardScripts/General/DeathPlane.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Numerics;
+using System.Threading.Tasks;
+using InfectedRose.Core;
+using InfectedRose.Lvl;
+using Uchu.Physics;
+using Uchu.World;
+using Uchu.World.Scripting.Native;
+
+namespace Uchu.StandardScripts.General
+{
+    /// <summary>
+    /// Native implementation of scripts/ai/act/l_act_player_death_trigger.lua
+    /// </summary>
+    [ScriptName("l_act_player_death_trigger.lua")]
+    public class DeathPlane : ObjectScript
+    {
+        /// <summary>
+        /// Creates the object script.
+        /// </summary>
+        /// <param name="gameObject">Game object to control with the script.</param>
+        public DeathPlane(GameObject gameObject) : base(gameObject)
+        {
+            //Console.WriteLine("Death barrier positioned at " + gameObject.Transform.Position + ", Scale of GameObject is " + gameObject.Transform.Scale);
```

Comment:
This probably should be removed. "Permanent" messages should use `Logger.Debug` or `Logger.Information` over `Console.WriteLine` for performance and thread safety reasons.

Reply:
I plan on removing this before it gets pushed to dev, I left it there in case I needed to use it again.